src/NHSD.BuyingCatalogue.Identity.Api/Infrastructure/TagHelperConstants.cs

```diff
@@ -38,8 +38,8 @@ public static class TagHelperConstants
         public const string NhsValidationSummaryTitle = "nhsuk-error-summary__title";
         public const string NhsValidationSummaryList = "nhsuk-error-summary__list";
         public const string NhsValidationInputError = "nhsuk-input--error";
-        public const string NhsBackLinkLink = "nhsuk-back-link__link";
-        public const string NhsIcon = "nhsuk-icon";
-        public const string NhsIconChevronLeft = "nhsuk-icon__chevron-left";
+        internal const string NhsBackLinkLink = "nhsuk-back-link__link";
+        internal const string NhsIcon = "nhsuk-icon";
+        internal const string NhsIconChevronLeft = "nhsuk-icon__chevron-left";
```

Comment:
I think it might be best if you make the class and all existing constants `internal` as well.

(The wording of my previous comment may have been less than fantastic!)

Reply:
No worries 👍